data-prepper-api/src/main/java/org/opensearch/dataprepper/model/log/JacksonOtelLog.java

```diff
@@ -109,7 +109,12 @@ public static JacksonOtelLog.Builder builder() {
 
     @Override
     public String toJsonString() {
-        final ObjectNode attributesNode = (ObjectNode) getJsonNode().get("attributes");
+        ObjectNode attributesNode;
```

Comment:
Try to avoid throwing exceptions when possible as they have a performance impact.

Also, we can avoid the `deepCopy()` in this situation.

```
Object anyAttributes = getJsonNode().get("attributes");
if(anyAttributes instanceof ObjectNode) {
    final ObjectNode flattenedJsonNode = getJsonNode().deepCopy();
    flattenedJsonNode.remove("attributes");
    // existing loop
    return flattenedJsonNode.toString();
}
return super.toJsonString();
```

Reply:
Thanks for the suggestion. This makes sense.